### PR TITLE
Game instructions

### PR DESCRIPTION
**Pull request by brianamarie**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/brianamarie/github-games/pull/4" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: http://githubschool.github.io/github-games/
 
+Here is how you play the game:
+1. Press the space bar to get started.
+2. Press the up arrow to rotate the shape.
```
Comment (brianamarie) [inline README.md]: commenting here

Comment (brianamarie) [inline README.md]: !fix this

Comment (brianamarie) [inline README.md]: !fixed this

**Pull request merged**